keyboards/ymd96/keymaps/default/keymap.c

```diff
@@ -24,7 +24,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define _DEFLT 0
 #define _RAISE 1
 
-#define KEYMAP KEYMAP_DEFAULT
+#define KEYMAP LAYOUT_DEFAULT
```

Comment:
Usually, we have everything after `LAYOUT` in lower case. 
```suggestion
#define KEYMAP LAYOUT_default
```